app/config.py

```diff
@@ -20,6 +21,13 @@
     extract_cloudfoundry_config()
 
 
+def str_to_bool(env_val) -> bool:
+    try:
+        return ast.literal_eval(env_val)
+    except Exception:
+        return False
```

Comment:
Thinking here... I think we might benefit from actually letting the exception raise instead of catching it. Wrong configuration value and interpretation for us, failing early could actually be helpful and avoid situations like the one we had this week.

Reply:
I added this because if you recall during mixpanal.. I did allow the exception to be thrown..

Reply:
Yeah, MixPanel is a case where we felt we didn't want the application to break if it was missing. In this case, a misconfiguration can be a production risk, so I would like this to fail early. Hope that makes sense!

Reply:
I had a chat with @patheard and @mohamed-cds today, especially around the lambda api and how it might crashes if we don't catch the exception and rely on defaults. The kubernetes pods have a fail safe where new versions won't get deployed if an error is detected at start up but the AWS lambda API won't have the same unfortunately. 

Hence we will revert to catching the exception and rely on a default, considering the unsafe aspect of switching AWS lambdas.

For other components such as the admin running uniquely in the kubernetes pods, the option to raise the exception and not catch it could make sense.

Reply:
I updated this to explicitly test for the String values of `"True"` or `"False"` and return the default value in all other cases.  

My reasoning was that in a case of calling `str_to_bool(os.getenv("NON_EXISTENT_KEY))`, the `str_to_bool` function would receive the value of `None` and `ast.literal_eval` would dutifully return this as `None` rather than the boolean value we're expecting.

Reply:
Alright. Can we though perform a `lower` and `strip` operation on the `str_val` parameter so the case does not matter?

Reply:
Absolutely - I'll get that added.

Reply:
Added!